### PR TITLE
MM-1782 makes the LHS dropdown close when clicking on the LHS header

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

**Comment by rgarmsen2295** [comment]
Looks good, +1

File: web/react/components/sidebar_header.jsx [hunk 1]
```diff
@@ -17,8 +17,15 @@ var NavbarDropdown = React.createClass({
         e.preventDefault();
         client.logout();
     },
+    blockToggle: false,
     componentDidMount: function() {
         UserStore.addTeamsChangeListener(this._onChange);
+
+        var self = this;
+        $(this.refs.dropdown.getDOMNode()).on('hide.bs.dropdown', function(e) {
```
Comment (nickago) [inline web/react/components/sidebar_header.jsx]: Do we need to have an 'off' event to complement this one?

Comment (jwilander) [inline web/react/components/sidebar_header.jsx]: Yes, good catch

Comment (jwilander) [inline web/react/components/sidebar_header.jsx]: Resolved with latest commit

**Comment by rgarmsen2295** [comment]
+1

**Pull request merged**